web/frontend/src/index.tsx

```diff
@@ -224,38 +224,42 @@ const AppContainer = () => {
     };
 
     async function fetchData() {
+      const response = await fetch(ENDPOINT_PERSONAL_INFO, req);
+      let result;
       try {
```

Comment:
Beware ! This is a dangerous change.
The `fetch` API returns a promise.
`await`ing on a Promise can throw an exception if the Promise is rejected.
This needs to be in the try/catch block.

Reply:
I changed this to put the try/catch around the whole function. What do you think?